### ISSUE TITLE
Export selected parts to a single file

### ISSUE THREAD

**Issue by Dima-S-Jr** [musescore/MuseScore#22995]
### Your idea

It would be cool if the user could export the selected parts to a single file (without resorting to workarounds), rather than each part to a single file (individually).

### Problem to be solved

This will allow users to export scores consisting of the parts they need (in a single file). 
(For example, I have a choir (SATB) and a piano. I only need to export SATB.)

Currently, you can export multiple parts to a single file only in PDF format. Recently, I had the need to export a bowed string section from the orchestra to an `mp3` file. Instead, MuseScore suggests exporting each part to a separate file.
![Снимок экрана (373)](https://github.com/musescore/MuseScore/assets/134041121/7719014d-fb44-49cc-a956-8d4cbefdd7f9)
![Снимок экрана (372)](https://github.com/musescore/MuseScore/assets/134041121/a6e5466e-ed9e-46ed-8a79-0cf2012b3752)

**Comment by MarcSabatella** [comment]
The way to do this is to create a part containing the instruments you want, then export it.  That’s not a “workaround” but the way the part system is intended to work.  The ability to easily define parts containing any combination of instruments you like is one of the major features of the system.

That’s not to say an option like this couldn’t be useful for some special cases also, though.

**Comment by MarcSabatella** [comment]
It would also create interesting issues if you had two parts defined, one with just woodwinds, the other with woodwinds + brass, then asked to have them “combined”.

**Comment by Jojo-Schmitz** [comment]
Back in the day #7500 implented something like this

**Comment by MarcSabatella** [comment]
I don’t see that #7500 deals with audio at all?

**Comment by Dima-S-Jr** [comment]
@cbjeukendrup, I don't quite understand your assessment of my request. The need to export some specific parts from the entire score to a single file may be caused for technical reasons (for example, I need to demonstrate to someone how the combos from a string band sound, or I need to listen only to percussion to analyze the sound quality).

Yes, I can do this by disabling the rest of the parts in the Mixer, but, you must agree, after all, this is a workaround.

**Comment by MarcSabatella** [comment]
No one said anything about the mixer.  The way to do it is as I already said - define a part containing the instruments you want.

**Comment by Dima-S-Jr** [comment]
> No one said anything about the mixer. 

See above (@cbjeukendrup's quote):

> It would make sense to include multiple tracks in an audio file, but as Marc says, that is already possible, by using either the solo/mute buttons in the mixer, or by creating a part score containing the tracks that you want to have exported.


> The way to do it is as I already said - define a part containing the instruments you want and export that.

Maybe we misunderstood each other. I need to select several specific parts from the full score and export them to a single sound file. Currently, I can export only one pert to a single audio file, not several. If I misunderstood you, then please correct me.

**Comment by MarcSabatella** [comment]
Yes, I understand exactly what you are wanting to do, which is why I explained how to to do it.  Simply create a part that includes the instruments you wish to export (using the Parts and Instruments panels as usual).  Then you can easily export that part at any time.  Again, the ability to easily define parts containing multiple instruments is an integral part of the design of the parts feature, not a “workaround” at all but exactly how it is designed to work.  it’s true the mixer can *also* be used, if you want, but it’s certainly;y not the way I was recommending.
.
As always, when you have questions about how to use MuseSocre, please ask in the official Support forum on musescore.org.

**Comment by cbjeukendrup** [comment]
@Dima-S-Jr Note that I'm not saying that the need to export an audio file with a certain set of tracks is not valid. I'm saying that this exact request is invalid, basically because it's the 5 millionth time that someone confuses part _scores_ with _instruments/players/tracks_. 

The list you see on the left in the export dialog, is a list of part scores, corresponding to the content of the Parts dialog. Semantically it doesn't make sense to combine multiple part scores for audio playback. As I explained, what does make sense, is a way to export selected _tracks_ together. And that is exactly what the Mixer is for, or you can create a part score containing exactly the tracks that you want, so that you don't need to select them in the mixer every time. 

If this is really not enough for you, we can put another list in the export dialog where you can select a custom list of tracks to export, but that really just duplicates the mixer.
